zrml/global-disputes/Cargo.toml

```diff
@@ -9,6 +9,9 @@ sp-std = { branch = "polkadot-v0.9.26", default-features = false, git = "https:/
 zeitgeist-primitives = { default-features = false, path = "../../primitives" }
 zrml-market-commons = { default-features = false, path = "../market-commons" }
 
+# Benchmark
+num-traits = "0.2.15"
```

Comment:
Just a suggestion, can we make this optional and enable it only if benchamark feature is enabled ?

Reply:
ffef740d5548692e6f5cdf12e4db9b64ef92ed10